src/molecules/categoriesNavbar/CategoriesNavbar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Category from "../../atoms/category";
```

Comment:
these atoms are specific to CategoryNavbar component, you can create these inside categoriesNavbar folder